src/vsd/meldingen/import/import.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+set -e
+source ${SHARED_DIR}/import/config.sh
+source ${SHARED_DIR}/import/before.sh
+
+ENVIRONMENT=${DATAPUNT_ENVIRONMENT:-acceptance}
+
+echo "Process import data"
+ZIPFILE=mora.zip
+OBJECTSTORE_PATH=Dataservices/meldingen/${ENVIRONMENT}/$ZIPFILE
+
+mkdir -p ${TMPDIR}/meldingen
+echo "Download file from objectstore"
+python $SHARED_DIR/utils/get_objectstore_file.py "$OBJECTSTORE_PATH" -s objectstore_dataservices
+
+unzip $TMPDIR/meldingen/${ENVIRONMENT}/$ZIPFILE -d ${TMPDIR}
+
+psql -X --set ON_ERROR_STOP=on << SQL
+DROP TABLE IF EXISTS pte.mora_melding_dag_buurt_categorie CASCADE;
+CREATE SCHEMA IF NOT EXISTS pte;
+\i ${TMPDIR}/mora.backup
+SQL
+
+psql -X --set ON_ERROR_STOP=on << SQL
+CREATE SEQUENCE IF NOT EXISTS public.meldingen_statistiek_seq;
+CREATE TABLE IF NOT EXISTS public.meldingen_statistiek (
+  id integer NOT NULL DEFAULT nextval('meldingen_statistiek_seq'),
```

Comment:
Alternatief is om een identity constructie te gebruiken. Dan hoef je geen sequence te maken. Zoals:

ID integer NOT NULL GENERATED ALWAYS AS IDENTITY,

Reply:
Ik kende deze nog niet. Goed om te weten. In dit geval wilde ik bij na truncate de sequence resetten. Ik zag niet zo snel hie dat met de  GENERATED identity moet.